tests/cephfs/clients/mds_scrub_after_failover.py

```diff
@@ -0,0 +1,144 @@
+import json
+import random
+import string
+import time
+import traceback
+
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from tests.io.fs_io import fs_io
+from utility.log import Log
+
+log = Log(__name__)
+
+"""
+Test steps for 83573489
+1. Fill up the cluster with 60% data
+2. Prepare the clients with different mounting options
+3. Set max mds to 2
+4. Set standby count to 2
+5. Start IO on all the clients
+6. Kill one of the active MDS
+7. Start scrubbing
+8. check if it shows any issue while scrubbing
+"""
+
+
+def run(ceph_cluster, **kw):
+    try:
+        fs_util = FsUtils(ceph_cluster)
+
+        config = kw.get("config")
+
+        clients = ceph_cluster.get_ceph_objects("client")
+
+        build = config.get("build", config.get("rhbuild"))
+
+        fs_util.prepare_clients(clients, build)
+
+        fs_util.auth_list(clients)
+
+        log.info("checking Pre-requisites")
+
+        client1 = clients[0]
+
+        cephfs = {
+            "fill_data": 60,
+            "io_tool": "smallfile",
+            "mount": "fuse",
+            "filesystem": "cephfs",
+            "mount_dir": "/mnt/mycephfs1",
+        }
+        # fill up to 60% of the cluster
+        fs_io(client=clients[0], fs_config=cephfs, fs_util=fs_util)
+        # Prepare the clients with different mounting options
+        # set max mds 2
+        client1.exec_command(sudo=True, cmd="ceph fs set cephfs max_mds 2")
+        # set standby count to 2
+        client1.exec_command(sudo=True, cmd="ceph fs set cephfs standby_count_wanted 2")
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(10))
+        )
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        kernel_mounting_dir_2 = f"/mnt/cephfs_kernel{mounting_dir}_2/"
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+        fuse_mounting_dir_2 = f"/mnt/cephfs_fuse{mounting_dir}_2/"
+        mon_node_ips = fs_util.get_mon_node_ips()
+
+        default_fs = "cephfs"
+
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mounting_dir_1,
+            extra_params=f" --client_fs {default_fs}",
+        )
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mounting_dir_2,
+            extra_params=f" --client_fs {default_fs}",
+        )
```

Comment:
can we combine these with for 'for loop' ?
for mount_dir in [fuse_mounting_dir_1,fuse_mounting_dir_2]:
     fs_util.fuse_mount(
            [client1],
            mount_dir,
            extra_params=f" --client_fs {default_fs}",
        )

Reply:
sure.